JS/inject.js

```diff
@@ -1,147 +1,150 @@
 class MessageData {
     constructor(fullContent = "a,b,c") { //signal, service, content
         var fullContent = fullContent.split(",");
-        this.signal = fullContent[0];
+        this.signal = fullContent[0]; //signal should be "studio721" to signify the message's purpose
         this.service = fullContent[1];
         this.content = fullContent[2];
     }
 }
 
+var lastSentVideo = "";
 var player,
     time_update_interval = 0;
-document.querySelectorAll(".l4V7wb")[0].addEventListener("mouseup", function () {
+
+//important class names
+var joinBTNClass = "l4V7wb"; //button that is clicked to join the meets call
+var addPeopleClass = "K74C9e"; //"add person" label in the chat/people menu that signifies that the user can add people 
+var meetingDetailsClass = "anXpBf"; //row of elements inside the meeting details tab for "People" and "Chat" tabs
+var buttonBarClass = "fT3JUc"; //bottom bar of the meets call that sometimes disappears
+var personNameClass = "NkoVdd"; //text label of name of person(s) inside meeting, viewable inside the meeting details -> people 
+
```

Comment:
Again, not asking for a change, just for your perspective as the developer.

Would it have made sense to have these stored in an external file and read in as key value pairs? This would mean not changing code if the classes change (and possibly storing versions to match different versions of meet?)